lib/galaxy/version.py

```diff
@@ -1,3 +1,3 @@
 VERSION_MAJOR = "24.0"
-VERSION_MINOR = "dev0"
+VERSION_MINOR = "rc1"
```

Comment:
A heads up, we've never done rc's before so this will likely require additional changes.